src/mlpack/methods/ann/layer/add_merge.hpp

```diff
@@ -107,15 +114,33 @@ class AddMerge
   //! Modify the delta.
   OutputDataType& Delta() { return delta; }
 
+  //! Return the model modules.
+  // std::vector<LayerTypes<CustomLayers...> >& Model()
+  // {
+  //   if (model)
+  //   {
+  //     return network;
+  //   }
+
+  //   return empty;
+  // }
```

Comment:
I think this commented code could be removed, but I'm not sure.  I guess maybe the layer should have a `Model()` function?